pyproject.toml

```
@@ -0,0 +1,129 @@
+[build-system]
+requires = ["setuptools>=61.0.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "django-digid-eherkenning"
+version = "0.15.0"
+description = "A Django app for DigiD/eHerkenning authentication flows"
+authors = [
+    {name = "Maykin Media", email = "support@maykinmedia.nl"}
+]
+readme = "README.rst"
+license = {file = "LICENSE"}
+keywords = ["django", "authentication", "digid", "eherkenning", "eidas", "dutch", "nl", "netherlands"]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Framework :: Django",
+    "Framework :: Django :: 4.2",
+    "Intended Audience :: Developers",
+    "Operating System :: Unix",
+    "Operating System :: MacOS",
+    "Operating System :: Microsoft :: Windows",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+]
+requires-python = ">=3.10"
+dependencies = [
+    "cryptography>=40.0.0",
+    "django>=4.2.0",
+    "django-sessionprofile",
+    "django-simple-certmanager",
+    "django-solo",
+    "lxml>=4.7.1",
+    "furl",
+    "maykin-python3-saml>=1.16.0",
+]
+
+[project.urls]
+Homepage = "https://github.com/maykinmedia/django-digid-eherkenning"
+Documentation = "https://django-digid-eherkenning.readthedocs.io/en/latest/"
+"Bug Tracker" = "https://github.com/maykinmedia/django-digid-eherkenning/issues"
+"Source Code" = "https://github.com/maykinmedia/django-digid-eherkenning"
+Changelog = "https://github.com/maykinmedia/django-digid-eherkenning/blob/master/docs/CHANGELOG.rst"
```

this is a dead link, it should be

```suggestion
Changelog = "https://github.com/maykinmedia/django-digid-eherkenning/blob/master/CHANGELOG.rst"
```